### ISSUE TITLE
Snapclient systemd fails sometimes

### ISSUE THREAD

**Issue by gerroon** [badaix/snapcast#652]
Hi

It seems like on this particular pc the systemd service for snapclient fails after a long period of inactivity between the snapclient and snapserver

I can start snapclient using systemd `systemctl start snapclient` after the failed restart below. So I am not sure what the issue could be.

I am on Debian testing using Snapclient v0.20 against Snapserver v0.20, both are on Debian Testing

```

 systemctl status snapclient.service 
● snapclient.service - Snapcast client
     Loaded: loaded (/lib/systemd/system/snapclient.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Sun 2020-06-28 20:15:09 CDT; 14h ago
       Docs: man:snapclient(1)
    Process: 307412 ExecStart=/usr/bin/snapclient --logsink=system $SNAPCLIENT_OPTS (code=exited, status=1/FAILURE)
   Main PID: 307412 (code=exited, status=1/FAILURE)

Jun 28 20:15:09 PC systemd[1]: snapclient.service: Main process exited, code=exited, status=1/FAILURE
Jun 28 20:15:09 PC systemd[1]: snapclient.service: Failed with result 'exit-code'.
Jun 28 20:15:09 PC systemd[1]: snapclient.service: Scheduled restart job, restart counter is at 12.
Jun 28 20:15:09 PC systemd[1]: Stopped Snapcast client.
Jun 28 20:15:09 PC systemd[1]: snapclient.service: Start request repeated too quickly.
Jun 28 20:15:09 PC systemd[1]: snapclient.service: Failed with result 'exit-code'.
Jun 28 20:15:09 PC systemd[1]: Failed to start Snapcast client.

```

**Comment by gerroon** [comment]
I wonder if this is related to this?

```

2020-06-30 00-49-25.136 [Info] (Connection) My MAC: "XXXXXX", socket: 8                                                          [5/774]
2020-06-30 00-49-25.615 [Info] (Controller) ServerSettings - buffer: 1000, latency: 0, volume: 40, muted: 0                                        
2020-06-30 00-49-25.617 [Info] (Controller) Codec: flac, sampleformat: 48000:16:2                                                                  
2020-06-30 00-49-25.617 [Info] (Player) Player name: alsa, device: default:CARD=PCH, description: HDA Intel PCH, CX20751/2 Analog                  
2020-06-30 00-49-25.617 [Info] (Player) Default Audio Device, idx: 15, sharing mode: unspecified                                                   
2020-06-30 00-49-25.617 [Info] (Player) Mixer mode: software, parameters: <none>                                                                   
2020-06-30 00-49-25.617 [Info] (Player) Sampleformat: 48000:16:2, stream: 48000:16:2
2020-06-30 00-49-25.634 [Fatal] (Snapclient) Exception: Can't open default:CARD=PCH, error: Device or resource busy
2020-06-30 00-49-25.634 [Notice] (Snapclient) daemon terminated.


```

**Comment by badaix** [comment]
this looks like a good bet for your problem. You should see these lines also in your syslog when systemd fails to start snapclient.
Please try to add 
`ExecStartPre=/bin/sleep 5`
in the `[Service]` section of `/lib/systemd/system/snapclient.service`

**Comment by gerroon** [comment]
Thanks I will try it.

**Comment by badaix** [comment]
Fixed in 9e06e52193e213fccf59434fed1d4311c8f9d9f5
